Sources/Misc/StoreKit2Setting.swift

```diff
@@ -38,6 +38,27 @@ extension StoreKit2Setting {
 
 }
 
+extension StoreKit2Setting {
+
+    /// - Returns: `true` if SK2 is available in this device.
+    static var isStoreKit2Available: Bool {
+        if #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *) {
+            return true
+        } else {
+            return false
+        }
```

Comment:
Could this work?
```suggestion
        return #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
```

Reply:
I'm pretty sure it doesn't unfortunately but let me try it

Reply:
Confirmed you can't. I hope Swift makes all this stuff expressions in the future.